Library/Homebrew/test/cmd/info_spec.rb

```diff
@@ -18,6 +18,13 @@
       .and not_to_output.to_stderr
       .and be_a_success
   end
+
+  it "looks for a Cask with the same name if a given Formula does not exist", :needs_network do
```

Comment:
This is failing `brew tests --generic --online`. Cask is macOS-only so the test will `:needs_macos` since it shouldn't be run on a generic OS.

Reply:
Thanks so much for this!